### PR TITLE
Article: Running Condor with NCTL

### PR DESCRIPTION
**Pull request by sczembor**
Guide explaining step by step how to run dockerized NCTL with Condor + compatible casper-client-rs

answers: #4

### CODE REVIEW

**Review by asladeofgreen — COMMENTED** [review]
0.  There's not much real Condor content here.
1. Couldn't we pivot to using CCTL rather than NCTL ?  
2. There are multiple SDKs: py, java, .net, rs, js ... so just mention utilisation via "a supported SDK".
3. The casper client tool should in my view be deprecated.  But if you are going to use it for simple illustration purposes then OK.
4. When running the NCTL network you should be able to use nctl-net-status command to view running status of the node and their sidecars.  Might be of use.

File: articles/004-local-setup.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+6. **Start the NCTL Docker Container:**
+   * **Docker Compose (Recommended):** If you're using the `docker-compose.yml` file, update the `image` line under the `mynctl` service to `casper-nctl:feat-2.0`. Then run `docker-compose up`.
+   * **Manual Docker Command:** 
+      ```bash
+      docker run -d --name mynctl -p 11101:11101 casper-nctl:feat-2.0
+      ```
+
+## Part 2: Casper Client (Rust)
+
+To interact with your local Condor network, we'll use the Casper Client written in Rust. This is just one option. There are versions of the JS and .NET SDKs that are also becoming compatible with Condor.
```
Comment (sczembor) [inline articles/004-local-setup.md:57]:
```suggestion
To interact with your local Condor network, we'll use the Casper Client written in Rust. This is just one option. There are other supported SDKs that are also becoming compatible with Condor.
```

File: articles/004-local-setup.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+   ```
+
+2. **Switch to the Condor-Compatible Branch:**
+   ```bash
+   git checkout feat-track-node-2.0
+   ```
+
+3. **Test Your Setup:**
+   ```bash
+   cargo run --release get-state-root-hash --node-address http://127.0.0.1:11101
```
Comment (sczembor) [inline articles/004-local-setup.md:72]:
```suggestion
   cargo run --release get-node-status --node-address http://127.0.0.1:11101
```

File: articles/004-local-setup.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+   ```
+   This command should return the current state root hash of your local network, indicating a successful connection.
```
Comment (sczembor) [inline articles/004-local-setup.md:74]:
```suggestion
   This command should return the node status your local network, indicating a successful connection. The output should look similar to this:
   ```
{
  "jsonrpc": "2.0",
  "id": -1220974167166441457,
  "result": {
    "api_version": "2.0.0",
.
.
.
```
```

File: articles/004-local-setup.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+```json
+{
+  "jsonrpc": "2.0",
+  "id": -1220974167166441457,
+  "result": {
+    "api_version": "2.0.0",
+  ...
+}
+
+## Using the Casper Client
```
Comment (sczembor) [inline articles/004-local-setup.md:84]:
```suggestion
}
```

## Using the Casper Client
```

File: articles/004-local-setup.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+   This command should return the node status of your local network, indicating a successful setup. The output should look similar to this:
+```json
+{
+  "jsonrpc": "2.0",
+  "id": -1220974167166441457,
+  "result": {
+    "api_version": "2.0.0",
+  ...
+}
+
```
Comment (sczembor) [inline articles/004-local-setup.md:83]:
```suggestion
```json
{
  "jsonrpc": "2.0",
  "id": -1220974167166441457,
  "result": {
    "api_version": "2.0.0",
  ...
}
```
```

**Comment by sczembor** [comment]
@asladeofgreen  I added some of the suggestions to the article. As of pivoting towards CCTL I think it is for another article. I think using casper-client here is OK since these are some basic operations + out entire docs are based on casper-client and people are familiar with it. Please take another look at the PR

**Pull request merged**